dcs/mission.py

```diff
@@ -254,6 +257,11 @@ def loaddict(fname: str, mizfile: zipfile.ZipFile, reserved_files: List[str]) ->
                 mapresource_dict = loaddict('l10n/DEFAULT/mapResource', miz, reserved_files)
                 self.map_resource.load_from_dict(mapresource_dict, miz)
 
+            if 'UHF_RADIO/SETTINGS.lua' in miz.namelist():
```

Comment:
so is this SETTINGS.lua file always bound to the a10c presets?
I really don't know just asking if there could be more than just a10 preset within this file?

Reply:
Thanks for reviewing!

My understanding is it used to be A-10C and KA-50 only, until recently. See this comment I made on the PR:

> Ok I now have a better understanding of the A-10C presets, and how their architecture recently changed. This PR will change course and add support for airframe-individual presets.

Consider this PR paused for now. It will look different when it is done for individual airframes. It will still be these SETTINGS.lua files, but they will be in airframe-id subdirectories in the miz file.